internal/status/status.go

```diff
@@ -140,9 +140,10 @@ func (u *UpdateWriter) Send(update Update) {
 // isStatusEqual checks if two objects have equivalent status.
 //
 // Supported objects:
-//  GatewayClasses
-//  Gateway
-//  HTTPRoute
+//
+//	GatewayClasses
+//	Gateway
+//	HTTPRoute
```

Comment:
This is the result of `gofmt` 1.19, which is different than the `gofmt` 1.18 that we're still using. This should be in a separate Go 1.19 PR.

Plus, this should be formatted as a bullet-list, not a code-block (godoc 1.19 added support for bullet lists).